provider/kavenegar/kavenegar.go

```diff
@@ -0,0 +1,67 @@
+package kavenegar
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/messagebird/sachet"
+)
+
+// Retrieving required data from 'kavenegar' sections of config.yaml
+type Config struct {
+	APIToken     string   `yaml:"api_token"`
+	PhoneNumbers []string `yaml:"phone_numbers"`
+}
+
+// Creating the KaveNegar to contain provider data
+type KaveNegar struct {
+	Config
+	HTTPClient *http.Client // The HTTP client to send requests on
+}
+
+// KaveNegar creates and returns a new KaveNegar struct
+func NewKaveNegar(config Config) *KaveNegar {
+	return &KaveNegar{
+		config,
+		&http.Client{Timeout: time.Second * 20},
+	}
+}
+
+// Building the API and call the KaveNegar endpoint to send SMS to the configured receptor from config.yaml
+func (ns *KaveNegar) Send(message sachet.Message) error {
+	const str0 string = "https://api.kavenegar.com/v1/"
+	var str1 string = ns.APIToken
+	const str2 string = "/sms/send.json"
+	var result string = str0 + str1 + str2
+	var KaveNegarURL = result
```

Comment:
it would be clearer if you used `fmt.Sprintf` to build the `url`.

Reply:
Actually, since Im not a developer and its my first interaction with Go so used another provider (NowSMS) to build the new provider.
Can it be still committed or its pending to mentioned change?

Reply:
It's was a good idea to base your work on another provider! 

By looking at the code I'm not sure it's correct or not but I noticed this problem that we didn't see before on the other provider. 
It's a small change it would be good to do it.

Also you mentioned that you are not a developer... Did you manage to test this change? To actually send an SMS with kavenegar? This is more important as the proposed change.

Reply:
I will try to change the code later in my next release because im a somehow busy on some projects.

I have tested the code with all possible condition, send to 1 and 2 and 3 receptors to make sure it works in single, couple and multiple receivers
also tested with and without sender part in config.yaml to make sure that is working even if user does not have dedicated number on KaveNegar (if as I have commented in provider,  you leave the from: empty KaveNegar will send the message from default number of account)
actually, it is now working in our Prometheus infrastructure in production environment so its OK for current release to be merged with the project.

Reply:
Hey @marcelcorso
I just changed the URL build from old method to fmt.Sprint and tested the provider with new changes.
Thanks for comment!